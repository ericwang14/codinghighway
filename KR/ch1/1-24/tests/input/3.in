#include <stdio.h>

int main()/*let's include some invalid code*/{
	int a[100] = { 0 ];
	a[[4] = 59;
	printf("It's a string, so no (counting(should be made)]]]]]]]\"right?)))))))))))))))))))))))\"");
	return (((0));
}
