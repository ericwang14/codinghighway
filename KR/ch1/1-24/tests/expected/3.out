Syntax error on line 4. Unbalanced close char: ]
Prematurely reached end of file - expected 1 closing braces
Prematurely reached end of file - expected 1 closing brackets
Prematurely reached end of file - expected 1 closing parentheses
